app/models/mapping.rb

```diff
@@ -11,13 +11,13 @@ class Mapping < ActiveRecord::Base
 
   SUPPORTED_TYPES = %w(redirect archive unresolved)
 
-  acts_as_taggable
+  acts_as_ordered_taggable
   has_paper_trail :ignore => [:tag_list => Proc.new { |mapping|
     # tag_list appears to ActiveModel::Dirty/paper_trail to be an
     # ActsAsTaggableOn::TagList but the original value from the database is a
     # string. This means that it is "different" even though it isn't.
     # Comparing the stringified version avoids that problem.
-    mapping.tag_list_was == mapping.tag_list.to_s
+    mapping.tag_list_was == mapping.tag_list
```

Comment:
Maybe we shouldn't use acts_as_ordered_taggable if it makes testing easier, but we have to change a bunch of tests to make them pass again?